### DISCUSSION TITLE
Deploying to a Custom Domain after deploying to a docs.page/org/home initially

### DISCUSSION THREAD

**Discussion by jwafu** [invertase/docs.page#300] [Q&A] [answered]
When deploying a docs.pages by default it is hosted at docs.page/org/home - but when adding a custom domain the subsequent changes creates two links (here the subdomain is 'subs'): 

- subs.org.com/home
and 
- docs.page/subs/home

Is there any way to resolve this or best practices to handle this?

**Comment by Ehesp** [comment, marked as answer]
This should be handled automatically - if we detect that the incoming request (`docs.page/subs/home`) has a domain within the domains.json (`subs/home`), any URLs on the rendered page should resolve back to the domain.

For example:

1. This page renders, even though it has a domain: https://docs.page/invertase/melos
2. Click on any internal link on the page (e.g. Getting Started), the URL is now: https://melos.invertase.dev/getting-started

**Reply by jwafu** [reply]
Thank you so much @Ehesp !! Makes perfect sense just wanted to check.
